packages/core-web/src/webrtc/shared.ts

```diff
@@ -26,21 +26,17 @@ export function createPeerConnection(
   const RTCPeerConnectionConstructor = getRTCPeerConnectionConstructor();
 
   if (RTCPeerConnectionConstructor) {
-    // strip non-standard port number if present
-    const hostNoPort = host?.split(":")[0];
-
-    const iceServers = host
-      ? [
-          {
-            urls: `stun:${hostNoPort}`,
-          },
-          {
-            urls: `turn:${hostNoPort}`,
-            username: "livepeer",
-            credential: "livepeer",
-          },
-        ]
-      : [];
+    const iceServers = [
+      {
+        urls: [
+          "stun:stun.l.google.com:19302",
+          "stun:stun1.l.google.com:19302",
+          "stun:stun2.l.google.com:19302",
+          "stun:stun3.l.google.com:19302",
+          "stun:stun4.l.google.com:19302",
```

Comment:
Just checking, but will removing this existing STUN/TURN configuration break for anyone that is currently using Livepeer webrtc servers?

Also these google URIs all point to the same pool of servers ... if we want to have multiple STUN options, then provider diversity is the way to go. We can include someone else like cloudflare.

```suggestion
          "stun:stun.l.google.com:19302",
          "stun:stun.cloudflare.com",
```

```bash
[nix-shell:~/Desktop]$ dig +short stun.l.google.com A
74.125.250.129

[nix-shell:~/Desktop]$ dig +short stun1.l.google.com A
74.125.250.129

[nix-shell:~/Desktop]$ dig +short stun2.l.google.com A
74.125.250.129

[nix-shell:~/Desktop]$ dig +short stun3.l.google.com A
74.125.250.129

[nix-shell:~/Desktop]$ dig +short stun4.l.google.com A
74.125.250.129
```